src/interpreter.cpp

```diff
@@ -764,6 +764,7 @@ struct command_info cmd_info[] =
     { "practice" , POS_RESTING , do_practice , 1, 0 },
     { "prance"   , POS_STANDING, do_action   , 0, 0 },
     { "pray"     , POS_SITTING , do_action   , 0, 0 },
+    { "probe"    , POS_RESTING , do_examine  , 0, SCMD_PROBE },
```

Comment:
Do we need to also change the line for examine to SCMD_EXAMINE?

Reply:
SCMD_EXAMINE is already defined as 0, so there will be no mechanical difference, but that's a good idea for clarity's sake.

Reply:
Fixed in b4f56da3cbb45e4d8f059eee2f662464495eb414.